grass7/vector/v.class.mlR/v.class.mlR.html

```diff
@@ -22,6 +22,9 @@ <h2>DESCRIPTION</h2>
 of features, a text file (<em>classification_results</em>) or reclassed 
 raster maps (<em>classified_map</em>).
 
+<p>When using text file input, the training data should not contain an id 
+column. The object data should have the ids in the first column.
```

Comment:
The object data (i.e., full set of data to be classified) should have the ids in the first column.